content/docs/command-reference/dag.md

```diff
@@ -76,6 +77,15 @@ $ dvc exp show ...
   [DOT](<https://en.wikipedia.org/wiki/DOT_(graph_description_language)>)
   format. It can be passed to third party visualization utilities.
 
+- `--mermaid` - show DAG in [Mermaid](https://mermaid-js.github.io) format. It
+  can be passed to third party visualization utilities.
+
+- `--md` - show DAG in `--mermaid` format, wrapped inside a Markdown
+  [fenced code block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks).
+
+  This can be used to combine `dvc dag` with
```

Comment:
Can we please reorder them the same as we end up deciding in https://github.com/iterative/dvc.org/pull/3454#discussion_r862525475 ?

Reply:
Done